ext/intl/grapheme/grapheme_string.c

```diff
@@ -134,19 +135,20 @@ PHP_FUNCTION(grapheme_strpos)
 /* {{{ Find position of first occurrence of a string within another, ignoring case differences */
 PHP_FUNCTION(grapheme_stripos)
 {
-	char *haystack, *needle;
-	size_t haystack_len, needle_len;
+	char *haystack, *needle, *locale = "";
+	size_t haystack_len, needle_len, locale_len = 0;
 	const char *found;
 	zend_long loffset = 0;
 	int32_t offset = 0;
 	zend_long ret_pos;
 	int is_ascii;
 
-	ZEND_PARSE_PARAMETERS_START(2, 3)
+	ZEND_PARSE_PARAMETERS_START(2, 4)
 		Z_PARAM_STRING(haystack, haystack_len)
 		Z_PARAM_STRING(needle, needle_len)
 		Z_PARAM_OPTIONAL
 		Z_PARAM_LONG(loffset)
+		Z_PARAM_STRING_OR_NULL(locale, locale_len)
```

Comment:
can you tell what happens if locale_len == 0 ?

Reply:
I meant if `locale_len`, means `locale` is empty so in this case what happens down the line ? should we get a default locale in this case ?